hacking/README.md

```diff
@@ -17,7 +17,7 @@ and do not wish to install them from your operating system package manager, you
 can install them from pip
 
     $ easy_install pip               # if pip is not already available
```

Comment:
Unrelated, but wonder if we should ditch reference to easy_install, as pip now recommends `get_pip.py`...